frontend/test/metabase/scenarios/models/reproductions/22715-remapped-values-override-column-identifier.cy.spec.js

```diff
@@ -28,6 +26,10 @@ describe.skip("issue 22715 ", () => {
 
       // Let's go straight to the model metadata editor
       cy.visit(`/model/${id}/metadata`);
+      // Without this Cypress fails to remap the column because an element becomes detached from the DOM
+      cy.findByText(
+        "Use the tab key to navigate through settings and columns.",
```

Comment:
So it will basically wait until this is shown before trying to do anything, since it otherwise might not fully have loaded DOM?

Reply:
Yup.
It seems that this string appears when all other fields are ready. Without it I've encountered local failures quite frequently.